docs/courses/README.md

```diff
@@ -1,28 +1,104 @@
+
+---
+
+**Confused?**
+[Ask people](https://github.com/kantord/LibreLingo/discussions) on GitHub Discussions.
+
+---
+
+**Table of Contents:**
+- [Getting started](#getting-started)
+- [Basics](#basics)
+  - [Terminology](#terminology)
+  - [Exploring the courses](#exploring-courses)
+  - [Things to do for new contributors](#things-new-contributors)
+- [Courses currently open for contributions](#courses-open)
+  - [For English speakers](#courses-from-english)
+- [Caution](#caution)
+  - [Languages that don't use the Latin alphabet](#non-latin-alphabet)
+
+## Getting started
+
+- Totally new to LibreLingo? Head to the [Basics](#basics) section! It will give you a general understanding of the course system.
+- Want to edit or improve existing courses? Check out [this page about editing courses](editing-courses.md).
+- If you want to create new courses (typically suited for course designers or advanced users), go to [this page](creating-courses.md).
+
+
+## Basics
+
+### Terminology
+
+You'll need to have an idea on basic terminology used in LibreLingo:
+
+- A **Course** is a set of Modules. There is typically one course for each language pair. e.g. Spanish for English speakers, Japanese for Italian speakers, etc.
+- A **Module** is a set of Skills.
+- A **Skill** is a small unit of course material. In the web app, the user will typically practice a Skill at a time.
+
+These concepts represent a hierarchical structure to LibreLingo course material: `Courses > Modules > Skills`
+
+- The **target language** is the language the course is meant to teach.
+- The **source langauge** is a language that users of the course are assumed to know.
+
+
+<a id="exploring-courses"></a>
+### Exploring the courses
+
+To have a better understanding of how courses work, you can explore how the course files are organized on this repository.
+
+You will find courses in the `courses` directory of [LibreLingo's repository](https://github.com/kantord/LibreLingo/tree/main/courses). The course directory name is usually structured in `<destination language>-from-<target audience's language>`. e.g. `french-from-english` teaches French to English speakers. In each course you'll find modules. And in each module you'll find skills, which are saved as individual .yaml files.
+
+Now let's look into how they look on the frontend, to the user.
+
+Go to [LibreLingo web app](https://librelingo.app/). Courses are listed there. e.g. You see a "START LEARNING SPANISH" button. (**NOTE:** Depending on the state and completeness of the courses, all of them might not be shown there on the website.)
+
+When you enter a course page, for example the [Spanish course](https://librelingo.app/course/spanish-from-english/), you'll see headings like Basics, Introduction, Activities. These are modules. Each module contains a set of skills. Modules are merely there to serve the purpose of organizing skills. Think of them as groups of units or chapters.
```

Comment:
Instead of 

> Modules are merely there to serve the purpose of organizing skills.

Actually I think it's enough to write

> Modules serve the purpose of organizing skills.

Reply:
Yes. Maybe you are right. But I've written it like this to give the users comfort that they don't need to think about it that much and make it friendlier.

Usually when we talk face to face we tend to use more words to express what we mean, even if those words are not absolutely necessary. I thought using words like this would ease their understanding of it a little bit.

Reply:
> Usually when we talk face to face we tend to use more words to express what we mean

That's a good point.

How about:

> Modules organize skills into groups, which is their only purpose.

I think this is even more conversational in style, and uses simpler grammar. In the end it's not much shorter than the original phrase, but maybe easier to read

Reply:
Sounds more formal to me. But maybe it's just me. :)
No problem, I'll apply the change.